packages/sensors/example/lib/snake.dart

```diff
@@ -73,12 +73,14 @@ class SnakeState extends State<Snake> {
   void initState() {
     super.initState();
     accelerometerEvents.listen((AccelerometerEvent event) {
+      if (!mounted) return;
       setState(() {
         acceleration = event;
       });
     });
 
     Timer.periodic(const Duration(milliseconds: 200), (_) {
+      if (!mounted) return;
```

Comment:
You can actually replace the `_` with `timer` and then `cancel` the timer as well, when not mounted.